### PR TITLE
Defining RemoteStream.AAC Caps

### PR DESCRIPTION
**Pull request by daniel-jodlos**
This PR defines experimental Remote AAC Caps, meant for use with formats that carry Audio Specific Config outside of the main AAC bitstream

### CODE REVIEW

File: lib/membrane_aac_format/remote.ex [hunk 1]
```diff
@@ -0,0 +1,12 @@
+defmodule Membrane.RemoteStream.AAC do
+  @moduledoc """
+  Capabilities defining Remote AAC Stream with known audio decoder configuration
```
Comment (bblaszkow06) [inline lib/membrane_aac_format/remote.ex:3]: The struct in this module is not "capabilities". I know it has been confusing, but at some point (if you're interested, the discussion is here: https://github.com/membraneframework/membrane_core/issues/236) we decided to define this properly:
1. Caps = capabilities = a set of formats supported by a pad, defined when defining a pad
2. Format = a specific media format that is described by a struct containing its properties

tl;dr This is the format definition

File: lib/membrane_aac_format/remote.ex [hunk 2]
```diff
@@ -0,0 +1,12 @@
+defmodule Membrane.RemoteStream.AAC do
+  @moduledoc """
+  Format definition for Remote AAC Stream with known audio decoder configuration
```
Comment (bblaszkow06) [inline lib/membrane_aac_format/remote.ex:3]: Ok, one more point here. "remote AAC stream" is not a well-known term. I can only guess what it is.
1. It should not start with capital letters
2. You should describe what kind of streams they are (depayloaded from containers with decoder config retrieved from that container?)

**Review by bblaszkow06 — CHANGES_REQUESTED** [review]

**Review by bblaszkow06 — APPROVED** [review]
